Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Action/Dashboard/Overlay.hs

```diff
@@ -202,6 +203,7 @@ data ScheduleOverlay = ScheduleOverlay
     rescheduleInterval :: Maybe Seconds,
     condition :: DTMO.OverlayCondition,
     overlayKey :: Text,
+    vehicleCategory :: Maybe DVC.VehicleCategory,
```

Comment:
We need to create entries of Overlays?

Reply:
yes

Reply:
Since there would be different config for different category.